app/models/wakes/location.rb

```diff
@@ -13,7 +14,8 @@ def label
     path
   end
 
-  def url(protocol: 'http', host: ENV['DEFAULT_HOST'])
+  def url(protocol: 'http', host: nil)
+    host ||= self.host || ENV['DEFAULT_HOST']
```

Comment:
No, it makes sense, but I wanted to run the script and validate it for Facebook metrics that show up only for production. For staging and local host they show up as nil or 0. Which is why I used an explicit host.